### DISCUSSION TITLE
Velero backup failing for mongo db

### DISCUSSION THREAD

**Discussion by saxenashivi2013** [vmware-tanzu/velero#5980] [Community support Q&A]
Backup is configured for mongo and consul PVs but having issue in mongo database only. this is the error - "Velero backup logs velero-every-day-at-7-20230313070007 | grep error
time="2023-03-13T07:01:37Z" level=info msg="1 errors encountered backup up item" backup=velero/velero-every-day-at-7-20230313070007 logSource="pkg/backup/backup.go:413" name=polaris-mongodb-0
time="2023-03-13T07:01:37Z" level=error msg="Error backing up item" backup=velero/velero-every-day-at-7-20230313070007 error="pod volume backup failed: running Restic backup, stderr=: signal: killed" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/restic/backupper.go:199" error.function="github.com/vmware-tanzu/velero/pkg/restic.(*backupper).BackupPodVolumes" logSource="pkg/backup/backup.go:417" name=polaris-mongodb-0
time="2023-03-13T07:03:40Z" level=info msg="1 errors encountered backup up item" backup=velero/velero-every-day-at-7-20230313070007 logSource="pkg/backup/backup.go:413" name=polaris-mongodb-1
time="2023-03-13T07:03:40Z" level=error msg="Error backing up item" backup=velero/velero-every-day-at-7-20230313070007 error="pod volume backup failed: running Restic backup, stderr=: signal: killed" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/restic/backupper.go:199" error.function="github.com/vmware-tanzu/velero/pkg/restic.(*backupper).BackupPodVolumes" logSource="pkg/backup/backup.go:417" name=polaris-mongodb-1
time="2023-03-13T07:05:53Z" level=info msg="1 errors encountered backup up item" backup=velero/velero-every-day-at-7-20230313070007 logSource="pkg/backup/backup.go:413" name=polaris-mongodb-2
time="2023-03-13T07:05:53Z" level=error msg="Error backing up item" backup=velero/velero-every-day-at-7-20230313070007 error="pod volume backup failed: running Restic backup, stderr=: signal: killed" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/restic/backupper.go:199" error.function="github.com/vmware-tanzu/velero/pkg/restic.(*backupper).BackupPodVolumes" logSource="pkg/backup/backup.go:417" name=polaris-mongodb-2"

**Comment by Satsank** [comment]
You are probably running out of resources. what is the size of the DB? Consider adding more memory as a first step.
